packages/devtools_app/lib/src/screens/inspector_v2/inspector_screen_details_tab.dart

```diff
@@ -10,63 +10,32 @@ import 'package:flutter/material.dart';
 import '../../shared/analytics/analytics.dart' as ga;
 import '../../shared/analytics/constants.dart' as gac;
 import '../../shared/common_widgets.dart';
-import '../../shared/globals.dart';
-import '../../shared/preferences/preferences.dart';
 import '../../shared/primitives/blocking_action_mixin.dart';
-import '../../shared/ui/tab.dart';
 import 'inspector_controller.dart';
 import 'inspector_screen.dart';
 import 'layout_explorer/layout_explorer.dart';
 
 class InspectorDetails extends StatelessWidget {
   const InspectorDetails({
-    required this.detailsTree,
     required this.controller,
     super.key,
   });
 
-  final Widget detailsTree;
   final InspectorController controller;
 
   @override
   Widget build(BuildContext context) {
-    final tabs = [
-      (
-        tab: _buildTab(tabName: InspectorV2DetailsViewType.layoutExplorer.key),
-        tabView: LayoutExplorerTab(controller: controller),
-      ),
-      (
-        tab: _buildTab(
-          tabName: InspectorV2DetailsViewType.widgetDetailsTree.key,
-          trailing: InspectorExpandCollapseButtons(controller: controller),
```

Comment:
we should be able to delete the `InspectorExpandCollapseButtons` widget below

Reply:
good catch, done!